src/app/docs/avatar/avatar-docs.component.html

```diff
@@ -7,14 +7,36 @@
     The avatar component displays an image to identify a record.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-code-examples
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Show image"
+          propertyName="showImage"
+          [checked]="true"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+    <sky-avatar
+      [canChange]="true"
+      [name]="name"
```

Comment:
Probably doesn't matter, but should we have a more distinct name in our code example? Setting a property to a text value that is the same as the property seems like it could be slightly confusing, but no worries if you don't think this is a problem. I was just thinking that maybe changing this value to something like `constituent name` here and in the `.ts` file might be clearer at a glance.

Reply:
When you use the square brackets around the property (`[name]`) the value is actually an expression that gets evaluated, so in this case `name` isn't a text value, but its a property in the .`ts` file. We actually follow this pattern in all our repos, so I don't think it'll be a problem.